docs/Icons.md

```diff
@@ -72,28 +76,15 @@ import { Battery } from '@brightlayer-ui/react-native-progress-icons';
 
 Brightlayer UI components will pass these values to you for use in your component to achieve the correct appearance. It is your responsibility to apply them.
 
-## Wrapped Icon
```

Comment:
You have made same changes here https://github.com/etn-ccis/blui-react-native-component-library/pull/483/files#diff-b1b786098c4fa4ff2f6eea4aa4816179b09676a5b43c6b2f3c3f6920dc9bfe7bL76.